perceval/utils/logging/loggers.py

```diff
@@ -72,8 +77,15 @@ def error(self, msg: str, channel: exq_log.channel = DEFAULT_CHANNEL, exc_info=N
     def critical(self, msg: str, channel: exq_log.channel = DEFAULT_CHANNEL, exc_info=None):
         pass
 
+    def log_resources(self, my_dict: Dict):
+        """Log a dictionary as resources, meaning at log level info, in channel user and serialize by json module
+
```

Comment:
This docstring seems a bit confusing to me :( I do not quite follow what it logs.